crates/bevy_image/Cargo.toml

```diff
@@ -11,7 +11,7 @@ keywords = ["bevy"]
 [features]
 default = ["bevy_reflect"]
 
-bevy_reflect = ["dep:bevy_reflect", "bevy_math/bevy_reflect"]
+bevy_reflect = ["bevy_math/bevy_reflect"]
```

Comment:
Nit: Maybe a comment here? I think normally we use `bevy_reflect` features to enable the dependency, so the purpose of this feature might not be exactly clear. I think it's to opt into deriving all the reflection traits?

Alternatively we could rename the feature to `derive_reflect` or something.